storm-core/src/ui/public/js/vis.min.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * vis.js
+ * https://github.com/almende/vis
+ *
+ * A dynamic, browser-based visualization library.
+ *
+ * @version 4.16.1
+ * @date    2016-04-18
+ *
+ * @license
+ * Copyright (C) 2011-2016 Almende B.V, http://almende.com
+ *
+ * Vis.js is dual licensed under both
+ *
+ * * The Apache 2.0 License
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * and
+ *
+ * * The MIT License
+ *   http://opensource.org/licenses/MIT
+ *
+ * Vis.js may be distributed under either license.
```

Comment:
With this we need to make sure that we update LICENSE.txt to include this file and the license that we pick for it.

We probably also want to white list this and vis.min.css for rat, so it stops complaining about them.